src/metrics/whole_scene_coverage.py

```diff
@@ -0,0 +1,26 @@
+from src.core import Database
+from src.metrics.reduction_metric import ReductionMetric
+from src.metrics.utils import build_map, get_bounds
+
+
+class WholeSceneCoverage(ReductionMetric):
+    """
+    The method returns the ratio of
+    the number of points in the map from the reduced database after down sampling
+    to the number of points in the map from the original database after down sampling
+    """
+
+    def __init__(self, voxel_size: float):
+        """
+        Constructs WholeSceneCoverage reduction metric
+        :param voxel_size: Voxel size for down sampling
+        """
+        self.voxel_size = voxel_size
+
+    def evaluate(self, original_db: Database, filtered_db: Database) -> float:
```

Comment:
I think we can pass list of filtered databases here to speed up calculations, because building map from the whole scene can be done at once